.github/workflows/artifact-hub-linter.yaml

```diff
@@ -1,12 +1,13 @@
-name: artifact hub linter
-on:
-  pull_request:
-    branches: [ main ]
+#Disable this job until https://github.com/artifacthub/hub/issues/3691 is resolved
```

Comment:
Create an issue for it to be reenabled?

Reply:
https://github.com/kubevirt/kubevirt-tekton-tasks/issues/341